zebra/interface.c

```diff
@@ -1696,6 +1701,10 @@ static void if_show_description(struct vty *vty, vrf_id_t vrf_id)
 	vty_out(vty, "Interface       Status  Protocol  Description\n");
 	FOR_ALL_INTERFACES (vrf, ifp) {
 		int len;
+		struct zebra_if *zif;
+		bool intf_desc;
+
+		intf_desc = false;
```

Comment:
`bool intf_desc = false` ?

Reply:
intf_desc was being set to false once with the `bool intf_desc = false` why? I am not sure as that was a bit surprising too me.  So I just moved it down one line to ensure it was reset at the top of the loop every time.